packages/nx/src/migrations/update-14-0-6/remove-roots.spec.ts

```diff
@@ -45,4 +45,22 @@ describe('remove-roots >', () => {
       );
     });
   });
+
+  describe('projects with package.json configs', () => {
+    beforeEach(() => {
+      tree = createTreeWithEmptyWorkspace(2);
```

Comment:
You probably need a tree.remove('workspace.json'), since createTreeWithEmptyWorkspace creates workspace.json, and as such project inference isn't active.